src/js/apps/patients/worklist/worklist_app.js

```diff
@@ -282,4 +292,36 @@ export default App.extend({
 
     this.showChildView('tooltip', tooltipView);
   },
+  showEditActionsSidebar() {
+    const headerView = new BulkEditActionsHeaderView({
+      actions: this.getState().getSelected(this.collection),
```

Comment:
We're doing this `state.getSelected` in a few places but they're all downstream of the first time.  I wonder if `toggleMultiSelect` should just store the selected collection on `this` and pass it to the multibutton so it doesn't have to get it again.
Similarly I think this is _slightly_ simpler if you just pass it here as `collection:` then in the view's template it's `items.length` or `this.collection.length` if needed in the view code.